src/main/kotlin/io/zerobase/smarttracing/resources/Amazon.kt

```diff
@@ -0,0 +1,179 @@
+package io.zerobase.smarttracing.resources
+
+import io.zerobase.smarttracing.models.*
+import software.amazon.awssdk.core.exception.SdkException
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.ses.SesClient
+import javax.mail.Message
+import javax.mail.MessagingException
+import javax.mail.Session
+import javax.mail.internet.AddressException
+import javax.mail.internet.InternetAddress
+import javax.mail.internet.MimeMessage
+import javax.mail.internet.MimeMultipart
+import javax.mail.internet.MimeBodyPart
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.nio.ByteBuffer
+import java.util.Properties
+import software.amazon.awssdk.core.SdkBytes
+import software.amazon.awssdk.services.ses.model.SendRawEmailRequest
+import software.amazon.awssdk.services.ses.model.RawMessage
+import io.zerobase.smarttracing.AmazonConfig
+
+// Currently only implements the AmazonSES
+class Amazon(private val config: AmazonConfig) {
```

Comment:
no we don't want a class taht represents all of amazon. we want an abstraction over sending emails, that's all. if we start using other services in aws, we will write one for that service at that time

Reply:
also read up on the law of demeter. don't pass in the config when you need the fields. pass in the fields you need

Reply:
i thought yesterday we agreed that we wanted it to be an abstraction of the entire AWS services? but either way this is resolved now